src/routes/(inner)/docs/getting-started/ssr.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	import { useFloating } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
+</script>
+
+<!-- Reference -->
+<div bind:this={floating.elements.reference}>Reference</div>
```

Comment:
Let's remove the reference here as it's out of scope for what's being discussed.

Reply:
And if possible let's look as updating the CodeBlock to use Shiki's row highlight feature:

![image](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/59e87fac-d85f-4038-aa10-c58760034a48)

Reply:
Done:
```html
        <CodeBlock
            lang="bash"
            code="{`
npm install @skeletonlabs/floating-ui-svelte
# pnpm install @skeletonlabs/floating-ui-svelte
# yarn install @skeletonlabs/floating-ui-svelte
# bun install @skeletonlabs/floating-ui-svelte
        `}"
            highlight="{1}"
        />
```
Creates:
![image](https://github.com/skeletonlabs/floating-ui-svelte/assets/63101006/20f2ed22-3472-46c6-89a8-e136ec7ec975)

Note: `highlight` takes in an array (`[1, 2 ,3]`) or a single number (`1`)